src/style/ManageQuestions.css

```diff
@@ -61,6 +61,25 @@
     border: 0;
 }
 
+.questions__textarea {
+    width: 100%;
+    height: 210px;
```

Comment:
looks good, we just need to test how these set heights look on different screen sizes